mne/preprocessing/maxwell.py

```diff
@@ -0,0 +1,255 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Jussi Nurminen
+#
+# License: BSD (3-clause)
+
+#TODO: write in equation numbers from Samu's paper
```

Comment:
Please also add a comment at the top saying that much of the code was "adapted with permission from Jussi Nurminen to relicense as BSD from <insert other git repo url>", and ping Jussi in this PR so he can follow, too